android/src/main/java/com/sbugert/rnadmob/RNAdMobBannerViewManager.java

```diff
@@ -1,7 +1,7 @@
 package com.sbugert.rnadmob;
 
 import android.content.Context;
-import android.support.annotation.Nullable;
+import androidx.annotation.Nullable;
```

Comment:
Changing this would break backwards compatibility. But React Native projects have support for Jetifier, so adding `android.enableJetifier=true` to your `gradle.properties` file should fix it for you.